Gems/ROS2/Code/Source/Sensor/SensorConfiguration.cpp

```diff
@@ -37,6 +38,7 @@ namespace ROS2
                         "Toggle publishing for topic")
                     ->DataElement(
                         AZ::Edit::UIHandlers::Default, &SensorConfiguration::m_frequency, "Frequency", "Frequency of publishing [Hz]")
+                    ->Attribute(AZ::Edit::Attributes::Min, AZStd::numeric_limits<float>::epsilon())
```

Comment:
I guess "1" makes plenty of sense here, but perhaps we have sensors that wish to publish something less often. Could we set a rational limit such as once per day? @michalpelka what is your opinion?

Reply:
I wanted to only disallow the user to input 0 as a frequency, so I figured that the smallest possible float would suffice. It seems that the O3DE Engine allows 0.0000001 as the smallest possible number to be input. This would allow sensors to update every 115 days. For now, I'll change the minimum to once per day.

Reply:
I agree with @arturkamieniecki, we should be able to publish with a frequency smaller than 1 Hz. Eps is good choice.